docs/source/developer/how_to_extend.rst

```diff
@@ -96,6 +103,9 @@ Install (and add to the requirement of your project) openwisp-radius::
 
     pip install openwisp-radius
 
+.. note::
+    Use ``pip install openwisp-radius[saml]`` if you intend to use
+    `Single Sign-On (SAML) <../user/saml_login.html>`_. feature.
```

Comment:
punctuation on this line looks wrong, I think it should be:

to use the Single Sign-On (..) feature.